mempool/src/counters.rs

```diff
@@ -444,7 +444,7 @@ pub static DB_ERROR: Lazy<IntCounter> = Lazy::new(|| {
 /// broadcast to, summed across each of its networks
 static ACTIVE_UPSTREAM_PEERS_COUNT: Lazy<IntGaugeVec> = Lazy::new(|| {
     register_int_gauge_vec!(
-        "mempool_active_upstream_peers_count",
+        "aptos_mempool_active_upstream_peers_count",
```

Comment:
If you're renaming all of these, you'll need to update the dashboards, too, right?

Reply:
The dashboards don't work at all currently because they already use the "aptos_" prefix. Apparently missed in a global replace awhile back.

Reply:
Synced offline 😄